spec/spec_helper.rb

```diff
@@ -1,3 +1,6 @@
+require 'coveralls'
+Coveralls.wear!('rails')
```

Comment:
I think this will likely be removed once updated with master, but noting anyways just in case.